Automattic/jetpack
# Required Review: Ignore reviews with state "COMMENTED"

#### Changes proposed in this Pull Request:

`fetchReviewers` should ignore reviews with state COMMENTED.

To determine if a reviewer has a green check mark (which signifies that the reviewer has approved the change), it doesn't look at the reviewer's last review, it looks at the last non-comment review.

For example, if a reviewer posts a review with state APPROVED, then posts a review with state COMMENTED, the reviewer will still have a green check mark.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* Have a reviewer post 2 reviews and make sure the check passes or fails as expected.
  * APPROVED, COMMENTED -> pass
  * CHANGES_REQUESTED/DISMISSED, COMMENTED -> fail
  * ... and so on.
